### PR TITLE
Added product listing page

### PR DESCRIPTION
**Pull request by kaustubh-jsr**
Added product listing page

### CODE REVIEW

File: pages/product-listing.html [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+            <aside class="sidenav sidenav-filter-section">
+                <section class="sidenav-section">
+                    <div class="sidenav-section__header filter-header">
+                        Filters
+                        <button class="btn btn-outline-secondary">Clear all</button>
+                    </div>
+                    <div class="filter-type">
+                        <h5>Category</h5>
+                        <div class="input-checkbox">
+                            <input type="checkbox" name="fruits" id="" checked>
```
Comment (eakansh125) [inline pages/product-listing.html:58]: id can be removed if not needed

**Review by eakansh125 — APPROVED** [review]
id can be removed if not required.
A lot of flex has been used, you can create a utility class for it, but its not a mandate for now.
Can be merged from my side!

File: pages/product-listing.html [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+                            </header>
+    
+                            <div class="card--links">
+                                <button class="btn btn-primary"><i class="fas fa-shopping-cart"></i> Add to Cart</button>
+                                <button class="btn btn-outline-secondary">Add to Wishlist</button>
+                            </div>
+                        </div>
+                    </a>
+                    <a href="#" class="card product-card-vertical product-card">
+                        <div class="card-basic--image">
```
Comment (parvez-rashid) [inline pages/product-listing.html:243]: I may be wrong but class naming doesn't look consistent here. I assume you've tried following some naming convention.

**Review by parvez-rashid — APPROVED** [review]
UI Review: UI looks amazing, great work. Code-wise I have a comment below, have a look. Other than that, the code is well-structured and neat. Can be merged.

**Comment by kaustubh-jsr** [comment]
> id can be removed if not required. A lot of flex has been used, you can create a utility class for it, but its not a mandate for now. Can be merged from my side!

Yeah I noticed that, I will do that post submission while refactoring. Thanks for the suggestion.

File: pages/product-listing.html (hunk 2 above)
Comment (kaustubh-jsr) [reply pages/product-listing.html:243]: No you are absolutely correct, I will try refactoring them once I am done with the rest of the components.

**Pull request merged**